rhel/vex/updater.go

```diff
@@ -0,0 +1,160 @@
+package vex
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+var (
+	_ driver.Updater           = (*VEXUpdater)(nil)
+	_ driver.Configurable      = (*VEXUpdater)(nil)
+	_ driver.DeltaUpdater      = (*VEXUpdater)(nil)
+	_ driver.UpdaterSetFactory = (*Factory)(nil)
+	_ driver.Configurable      = (*Factory)(nil)
+)
+
+const (
+	// TODO: VEX data is currently in beta, update URL when VEX data is GA.
+	baseURL        = "https://access.redhat.com/security/data/csaf/beta/vex/"
```

Comment:
Looks like it's available now: https://access.redhat.com/security/data/csaf/v2/vex/